src/interfaces/ICozyLens.sol

```diff
@@ -1,29 +1,12 @@
 // SPDX-License-Identifier: Unlicensed
 pragma solidity ^0.8.0;
 
-import "src/interfaces/IDecayModel.sol";
-import "src/interfaces/IDripModel.sol";
+import "src/interfaces/ISet.sol";
 
 /**
  * @notice Helper contract for reading data from the Cozy protocol.
  */
 interface ICozyLens {
-  /// @notice  Set-level configuration.
-  struct SetConfig {
```

Comment:
inlining these structs here makes it weird for integrators - in some cases they'll have to cast `IConfig.SetConfig` to `ICozyLens.SetConfig` for example

Reply:
Can they be defined outside of the interface? Can't recall offhand if that works for structs or just functions/errors. For example:

```solidity
// IWhatever.sol
struct X {
  uint256 a;
}

interface Y {
  function foo(X x) external;
}
```

Then when you import the above file, it'd be available directly

Reply:
Ah I see these live in `IConfig` anyway, but same concept: could potentially have no `interface {` there so you don't need `IConfig.SetConfig` in other files

Reply:
[2496369](https://github.com/Cozy-Finance/cozy-v2-interfaces/pull/7/commits/2496369daa4d572e2bdab0a3dceaa4b221b83a12) this seems to work well, I tested by locally updated the dependency in cozy-v2-triggers as well